### ISSUE TITLE
Toggle off dynamic font sizing does not update the designer display

### ISSUE THREAD

**Issue by peteward** [pdfme/pdfme#191]
### Describe the bug

When you un-check dynamic font sizing the changes are not applied immediately. 

### To Reproduce

* Create a large text box
* Enter a small amount of sample text
* Check dynamic font size
* Increase the maximum font size
* The text should dynamically increase in the designer to the new size
* Uncheck dynamic font size
* The text does not return to it's original size

### Expected behavior

* The text should return to it's originally defined size when unchecked.

### Your Environment

```markdown
- pdfme package(@pdfme/ui):
- pdfme version: 1.2.0
- Operating system: Demo designer
- Node.js version or Browser name & version: Chrome 114.0.5735.106
```

### Your Error Log

```shell
-
```

### Additional context

We will contribute the fix for this.

**Comment by hand-dot** [comment]
This bug fix will be included in the next release.

**Comment by hand-dot** [comment]
Released as 1.2.6!
